src/runtime/utils/refreshHandler.ts

```diff
@@ -73,7 +73,7 @@ export class DefaultRefreshHandler implements RefreshHandler {
     // Listen for when the page is visible, if the user switches tabs
     // and makes our tab visible again, re-fetch the session, but only if
     // this feature is not disabled.
-    if (this.config?.enableOnWindowFocus && document.visibilityState === 'visible') {
+    if (this.config?.enableOnWindowFocus && document.visibilityState === 'visible' && this.auth?.status.value !== 'unauthenticated') {
```

Comment:
Above, the check to refresh periodically, is based on if `this.auth?.data.value` is set. Do you see any advantage in checking against the status? 

I think we should use the same check for both, to ensure consistency, so I would propose to either change this to check if the data is set, or change the check for `refreshPeriodically`, if you belive checking against status is better 😊

Reply:
@zoey-kaiser 
Hi zoey🫶
Thank you for your kind review! Yes you are right. I've confirmed how it works with using `data.value` in 4fd1da8f1452ab0e174b461ccd59edf32111a571 
please kindly confirm😊